diksha1159.json

```
@@ -0,0 +1,17 @@
+{
+  "name": "Diksha Jadhav",
+  "bio": "Just a CS student trying to learn new exciting stuff"
```

```suggestion
  "bio": "Just a CS student trying to learn new exciting stuff",
```

Include a `,` to parse the json correctly